independent-projects/bootstrap/maven-plugin/src/main/java/io/quarkus/maven/BuildDependencyTreeMojo.java

```diff
@@ -0,0 +1,76 @@
+/**
+ *
+ */
+package io.quarkus.maven;
+
+import java.util.List;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Component;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.apache.maven.project.MavenProject;
+import org.eclipse.aether.RepositorySystem;
+import org.eclipse.aether.RepositorySystemSession;
+import org.eclipse.aether.repository.RemoteRepository;
+import io.quarkus.bootstrap.model.AppArtifact;
+import io.quarkus.bootstrap.resolver.AppModelResolverException;
+import io.quarkus.bootstrap.resolver.BootstrapAppModelResolver;
+import io.quarkus.bootstrap.resolver.maven.MavenArtifactResolver;
+
+@Mojo(name = "build-dependency-tree", defaultPhase = LifecyclePhase.NONE, requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
```

Comment:
We could shorten the name no?

What about `build-tree` or even just `tree` as the Maven Dependency plugin?

Reply:
Thanks, that's why I wrote to the dev list. I think there has to be a hint that these are the dependencies used for the build, not runtime or something. (potentially, we could add goals to displays other classpath trees, e.g. https://gist.githubusercontent.com/dmlloyd/c1aaadb9635e12b23581137c4ba9b841/raw/class-loaders.txt)

The maven analogue is specialized on dependencies, i.e. 'dependency:tree', while `quarkus-bootstrap` is dedicated to anything related to bootstrap.

I guess, `build-tree` could work though. How about `build-cp`?

Reply:
`build-tree` looks good to me. Short and clear.

Reply:
Agreed